### ISSUE TITLE
Saving into other formats

### ISSUE THREAD

**Issue by ibinsnur** [jasp-stats/jasp-issues#611]
Hi, 

I have to work with SPSS a lot (workplace requirement). I do like the ease and lightness of JASP and use it because of that, but I miss the possibility to save changed .sav datasets or export into .xlsx format. 
The shortcoming of the .csv format is the import process back to SPSS, which requires a lot of manual adjustments. 
It's plausible, if it's left out on purpose, because of the open license philosophy, but I didn't find anything concerning save formats, so I thought I might as well ask. 

Thanks for the good work!

**Comment by FransMeerhoff** [comment]
HI @ibinsnur 

We've placed this issue on our backlog. We must admit that this issue has no very high priority at the moment.

Kind regards
Frans Meerhoff

**Comment by JorisGoosen** [comment]
This should however not be too hard with the [Readstat support](https://github.com/WizardMac/ReadStat) we added a while ago.

**Comment by EJWagenmakers** [comment]
If it's easy to do then I think we should do it :-)

**Comment by JorisGoosen** [comment]
Well it should be relatively easy for stats/sas/spss. As far as excel goes we should probably first implement reading those formats? And would we also want to support exporting to ods format? 

Right now, as far as I know, we only export to csv right?

**Comment by boutinb** [comment]
There a some free package that read/write xlsx Excel format. We can have a look.
It would be good anyway to extend our export feature: in different format, with or without computed columns, with or without filter...

**Comment by JorisGoosen** [comment]
Yeah and I remember seeing an issue for better excel read support but I can't find it anywhere.

**Comment by AlexanderLyNL** [comment]
This one? 

https://github.com/jasp-stats/INTERNAL-jasp/issues/412

**Comment by PsyTechMMU** [comment]
Reading .xlsx files would be very welcome. :)

**Comment by asjones987** [comment]
It wold be helpful if reading XLSX files was supported out of the box.  It sounds like some of the options are cross platform compatible.

https://www.marsja.se/r-excel-tutorial-how-to-read-and-write-xlsx-files-in-r/

http://www.sthda.com/english/wiki/r2excel-read-write-and-format-easily-excel-files-using-r-software

http://www.sthda.com/english/wiki/r-xlsx-package-a-quick-start-guide-to-manipulate-excel-files-in-r

**Comment by tomtomme** [comment]
Dear @ibinsnur, since some time now, you can save/export JASP data as ODS. You would then open the ODS in Excel and save it as XLSX. You would then load the XLSX in SPSS.  I hope this helps

**Comment by richlv** [comment]
Usecase for saving/exporting sav - collaborating with SPSS users.
Otherwise each conversion is a major pain (loss of labels etc), followed by suggestions to move to SPSS :)

**Comment by JorisGoosen** [comment]
Not that it isnt an interesting idea to have more export formats.

**Comment by TarandeepKang** [comment]
I also give a thumbs up for more file formats! XPORT files for SAS are in readstat which can also write/export most of the file types JASP can currently import.
for JAMOVI files: https://cran.r-project.org/web/packages/jmvReadWrite/index.html and for JSON you may like https://cran.r-project.org/web/packages/jsonlite/index.html On the forum and in various issues I recall seeing various non-psychological scientists asking about transitioning from GraphPad Prism. I wonder if it would therefore be a good idea to include import and export of this software's two file formats .pdfx and .prism? Perhaps using https://github.com/Biomiha/prism2R?

**Comment by tomtomme** [comment]
some progress now here: https://github.com/jasp-stats/jaspBase/pull/211
